cropper/app/lib/Crops.scala

```diff
@@ -26,14 +27,27 @@ object Crops {
     s"${source.id}/${Crop.getCropId(bounds)}/${if(isMaster) "master/" else ""}$outputWidth.${fileType}"
   }
 
-  def createMasterCrop(apiImage: SourceImage, sourceFile: File, crop: Crop, mediaType: MimeType, colourModel: Option[String]): Future[MasterCrop] = {
+  def createMasterCrop(apiImage: SourceImage, sourceFile: File, crop: Crop, mediaType: MimeType, colourModel: Option[String],
+                      colourType: String): Future[MasterCrop] = {
+
     val source   = crop.specification
     val metadata = apiImage.metadata
     val iccColourSpace = FileMetadataHelper.normalisedIccColourSpace(apiImage.fileMetadata)
 
     for {
       strip <- ImageOperations.cropImage(sourceFile, source.bounds, 100d, Config.tempDir, iccColourSpace, colourModel, mediaType.extension)
-      file  <- ImageOperations.appendMetadata(strip, metadata)
+      file: File <- ImageOperations.appendMetadata(strip, metadata)
+
+      //Before apps and frontend can handle PNG24s we need to pngquant PNG24 master crops
+      optimisedFile =  if (colourType == "True Color" || colourType == "True Color with Alpha") {
```

Comment:
Could it ever be `True Color`? I thought these would go through PNGs-that-should-really-be-JPGs route... Not a problem as such, though.